src/shadertools/modules/fp64/test/fp64-shader.spec.js

```diff
@@ -139,11 +138,11 @@ function initializeGL(canvas)
   });
   gl.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT);
 
-  var fp_texture_support = gl.getExtension('OES_texture_float');
-  if (!fp_texture_support)
-  {
-    console.error("no floating point texture support!");
-  }
+  // var fp_texture_support = gl.getExtension('OES_texture_float');
```

Comment:
Was the check removed due to WebGL2 not supporting this extension?

If so, an option is using luma.gl capability system to do this check, it will do the right thing both on WebGL 1 & 2.